### DISCUSSION TITLE
Add a URL parameter to query only maps that were edited since a specific timestamp

### DISCUSSION THREAD

**Discussion by itspangler** [allmaps/allmaps#108] [Ideas for Apps: Editor, Viewer, ...]
The API should include a URL param that lets a user query only maps that were edited since a specific timestamp. @bertspaan's idea - just posting it here so we don't forget!

**Comment by itspangler** [comment]
@bertspaan, is this resolved by the new API which introduces "created" and "modified" values to georef annotations? or should it still be active?

**Comment by bertspaan** [comment]
Still active! Haven't added query params that can search/filter by created and modified dates in the new API yet.
